packages/core/src/scss/components/bigcommerce/forms-ccFields/_forms-ccFields.scss

```diff
@@ -99,3 +99,21 @@
 .form-field--setAsDefaultInstrument {
     order: 7;
 }
+
+#paypalcommercecreditcards-ccNumber,
+#paypalcommercecreditcards-ccExpiry,
+#paypalcommercecreditcards-ccName,
+#paypalcommercecreditcards-ccCvv,
+#bigcommerce_payments_creditcards-ccNumber,
+#bigcommerce_payments_creditcards-ccExpiry,
+#bigcommerce_payments_creditcards-ccName,
+#bigcommerce_payments_creditcards-ccCvv {
+  height: 45px;
+  overflow: hidden;
```

Comment:
Inner PayPal component does not rely on parent component height and shows weird grey blink animation for component loading that is bigger then parent container. The loading animation creates pretty bad UX experience, so we need to cut it off